cloud/docker.go

```diff
@@ -110,7 +109,6 @@ func (m *dockerManager) SpawnHost(ctx context.Context, h *host.Host) (*host.Host
 		"message": "created and started Docker container",
 		"host":    h.Id,
 	})
-	event.LogHostStarted(h.Id)
```

Comment:
These are visible in the event log https://evergreen.mongodb.com/event_log/host/i-01a4047e69210780b. I don't follow the reason for removing them here. Could you explain? I think they are still potentially useful (to us, if not to users) because they tell us when `SpawnHost` has returned.

Reply:
The thinking was to factor out the logging from the individual cloud providers and log when the create host job is finished, after SpawnHost returns and the host is inserted in the db. The time should be around the same.

Reply:
I added the new events to the host event log pages.